.github/workflows/e2e-nvidia-a10g-x1.yml

```diff
@@ -154,7 +154,7 @@ jobs:
 
           python3.11 -m pip show nvidia-nccl-cu12
           
-          ./scripts/basic-workflow-tests.sh -emf
+          ./scripts/basic-workflow-tests.sh -emfT
```

Comment:
Looking at the [matrix](https://github.com/instructlab/instructlab/blob/main/docs/ci.md/) it looks like full training was already working on this instance type and we need this to be added to the `x4` variant of this job?

@russellb can you confirm?

Reply:
Right.

My preference would be to leave this workflow alone and to add `-T` to `a10g-x4.yml`.

The change here is giving `-f` and `-T` which still has it running both training types. It would be better if this workflow only used full linux legacy training after this PR.

Reply:
well actually `-T` overrides ``-f` it seems, but it still doesn't make sense to specify both.

and again, i'd leave `-f` here and add `-T` to the bigger one.

Reply:
removed from a10x1, added to a10x4